src/convert/transformers/decomposedMetadataTransformer.ts

```diff
@@ -68,36 +69,39 @@ export class DecomposedMetadataTransformer extends BaseMetadataTransformer {
             type: childType,
             parent: component,
           };
-          const source = new JsToXml({
-            [childType.name]: Object.assign({ [XML_NS_KEY]: XML_NS_URL }, value),
-          });
-          // if there's nothing to merge with, push write operation now to default location
-          if (!mergeWith) {
-            writeInfos.push({
-              source,
-              output: this.getDefaultOutput(childComponent),
-            });
-          }
-          // if the merge parent has a child that can be merged with, push write
-          // operation now and mark it as merged in the state
-          else if (childrenOfMergeComponent.has(childComponent)) {
-            const mergeChild: SourceComponent = childrenOfMergeComponent.getSourceComponents(childComponent).first();
-            writeInfos.push({
-              source,
-              output: mergeChild.xml,
+          // only process child types that aren't forceignored
+          if (forceIgnore.accepts(this.getDefaultOutput(childComponent))) {
```

Comment:
This all looks great.  What do you think of adding an `else` to this that will `debug.log` that a child component is being ignored?